### PR TITLE
Update/Add justify-content helper class

### PR DESCRIPTION
**Pull request by altaudio**
### 💬 Description
Add `justify-content` helper class.

Same idea as `align-items` helper class.

Also removes the old `space-between` helper class. Usage instances have been removed here and in app (as you will see in the required PR 👇)

Required for https://github.com/gathercontent/app/pull/2655
### 🔗 Links
_Links that relate to the PR (Trello, Figma etc.)_
### 📹 GIF (optional)
_A short GIF of the changes in action._
### 🚪 Start Points
_Where is a good place to start the review? If there are multiple changes it may be worth listing multiple files._
### 👫 Related PRs (optional)
_Any PRs that relate to the changes._

### ✅ Checklist
- [ ] Tests written
- [ ] Browser tested
- [ ] Added to documentation
- [ ] Added to storybook

### CODE REVIEW

File: lib/CheckToggle/index.js [hunk 1]
```diff
@@ -88,6 +88,7 @@ class CheckToggle extends Component {
         {labelLeft && <p className={wrapperLabelClasses}>{labelLeft}</p>}
         <span className="toggle-wrapper__inner">
           <input
+            data-testid={id}
```
Comment (AmeeMorris) [inline lib/CheckToggle/index.js]: Do we need this? Cant we just grab this by the label text?

**Comment by altaudio** [comment]
[Add CTA to the bottom of the Shareable menu](https://trello.com/c/iYXTeWXb/1106-add-cta-to-the-bottom-of-the-shareable-menu)

File: lib/CheckToggle/index.js (hunk 1 above)
Comment (altaudio) [reply lib/CheckToggle/index.js]: Good point, we should be able to!

Comment (altaudio) [reply lib/CheckToggle/index.js]: > Do we need this? Cant we just grab this by the label text?

To follow up on this, the `CheckToggle` component basically contains debt that means we cannot target it `byLabelText`.

The current `CheckToggle` component uses a `<label>` element as the actual switch (and hides the native input), and uses `<p>` tags for the labels.

I took a look at the refactor and I think it is going to go our the scope of this PR. One for the future though under lesser time pressures 👍

**Pull request merged**